src/ophys_etl/transforms/mesoscope_2p/tiff.py

```diff
@@ -330,6 +355,8 @@ def volume_views(self):
         if self._volumes is None:
             self._volumes = []
             page_offset = 0
+            # this logic is probably incorrect if there are 2+ volume scans
+            # with identical zs, but needs testing to confirm
```

Comment:
Is there a conclusion to the testing suggested by this comment?

Reply:
Similar to my response to Dan above:

Natalia says that the operators should not be producing such data, but it is possible. The data model is poorly documented enough that I did not feel confident artificially constructing such a dataset (I cannot tell what it should look or behave like). Natalia offered to generate some real data under this case, if we want to pursue this further. That is probably something to worry about after the data release is under control.

So: no. We did not test this case.